lib/src/indicators/slider.dart

```diff
@@ -13,14 +13,17 @@ const double _kTickHeight = 8.0;
 const double _kDiscreteThumbWidth = 6.0;
 const double _kDiscreteThumbBorderRadius = 8;
 
+/// {@template macosSlider}
 /// A slider is a horizontal track with a control, called a thumb,
 /// that people can adjust between a minimum and maximum value.
 ///
 /// The slider doesn't maintain any state itself, instead the user is expected to
 /// update this widget with a new [value] whenever the slider changes.
 ///
-/// ![Slider](https://developer.apple.com/design/human-interface-guidelines/images/intro/components/slider-intro-dark_2x.png
+/// ![Slider](https://developer.apple.com/design/human-interface-guidelines/images/intro/components/slider-intro-dark_2x.png)
```

Comment:
```suggestion
/// {@image <image src="https://developer.apple.com/design/human-interface-guidelines/images/intro/components/slider-intro-dark_2x.png" height="150"/>}
```
Let's do it like this, otherwise the image is enormous